helpers/calculateEndDate.js

```diff
@@ -4,11 +4,13 @@ var moment = require('moment');
 
 module.exports = function(dust) {
 
-  /*
+/*
```

Comment:
Think it should be:

```
/**
 *
 */
```
(no spaces in front of the first `/` and one space in front of the `*`) as per http://usejsdoc.org/